logic.js

```diff
@@ -123,16 +123,19 @@ async function reduceFileDatum (fileDatum, address) {
   API.log(`[+] Checking Exist Record`, API.logLevel.INFO)
   for (var fileData of fileDatum) {
     API.log(` - Checking ${fileData.dKey}`, API.logLevel.INFO)
-    var fileTX = await API.findExist(fileData.buf, fileData.mime)
```

Comment:
Then we don't need to change this

Reply:
This is changed so that all TX ids can be compared, when looking for D records.  Not doing that is producing the edge cases that result in uploading repeatedly forever.  But I see I failed to make the same change to findD, leaving more edge cases.  I'll close this PR and reopen it when that change is made.

Reply:
Update:  Actually, it looks good.  The reason this is changed is so that all the TX ids can be compared with the most recent D record.  If only one is compared, it reuploads again.  (I also had an additional bug where it would reupload forever; I haven't found that one yet.)